samples/C/asm.h

```diff
@@ -0,0 +1,27 @@
+/* CarbonOS System/Kernel
+ * Copyright 2015-2017 David Aylaian
+ * Licensed under GitHub CLA: https://cla.github.com/agreement
```

Comment:
Well, that works, but a [permissive license](https://github.com/github/linguist/blob/9289704/vendor/licenses/config.yml#L2-L11) (e.g., MIT/Expat-compatible) is really all we need. =) It is of course your personal choice, however.